.github/workflows/build-hw-reusable.yml

```diff
@@ -29,14 +29,18 @@ jobs:
     strategy:
       matrix:
         adapter: [
-          {name: "${{inputs.name}}", platform: "${{inputs.platform}}"},
+          {name: "${{inputs.name}}", platform: "${{inputs.platform}}", static_Loader: OFF},
         ]
         build_type: [Debug, Release]
         compiler: [{c: gcc, cxx: g++}, {c: clang, cxx: clang++}]
         # TODO: The latest L0 loader segfaults when built with clang.
         exclude:
          - adapter: {name: L0, platform: ""}
            compiler: {c: clang, cxx: clang++}
+        include:
+         - adapter: {name: L0, platform: "", static_Loader: ON}
```

Comment:
did you figure out how we can avoid having this for every adapter? I think that's the only thing blocking us from merging this.

Reply:
I've added a `static_loader` param to the `build-hw-reusable.yml` and a `level-zero-static` job that sets this param. Take a look.